integration-cli/docker_cli_run_test.go

```diff
@@ -2516,3 +2516,30 @@ func (s *DockerSuite) TestRunWriteFilteredProc(c *check.C) {
 		}
 	}
 }
+
+func (s *DockerSuite) TestRunNetworkFilesBindMount(c *check.C) {
+	name := "test-nwfiles-mount"
+
+	f, err := ioutil.TempFile("", name)
+	if err != nil {
```

Comment:
Given the comment about using `dockerCmd` (which was just done across the test codebase in very recently merged PRs) it might also be good to align with the slow migration to use the test framework `Assert` capabilities and get rid of the `if err != nil` clauses where applicable.  See a few tests "up" in the file for good examples of the nil check with `c.Assert`.